production/ksonnet/loki/distributor.libsonnet

```diff
@@ -47,6 +47,14 @@ local k = import 'ksonnet-util/kausal.libsonnet';
     else
       k.util.antiAffinity,
 
+  distributor_pdb:
+    local podDisruptionBudget = $.policy.v1.podDisruptionBudget;
```

Comment:
nit: should we use `$.policy.v1.podDisruptionBudget` for clarity? I see this usage in other files `$.policy.v1.podDisruptionBudget`, not sure how `ksonnet-util/kausal.libsonnet` contents  ended up being at top-level